### DISCUSSION TITLE
[[z:zettels?]] extension

### DISCUSSION THREAD

**Discussion by GenrihGrigoryan** [srid/neuron#492] [Ideas]
It would be great, to add more possibilities to [[z:zettels?]]:

1) Multiple tag: [[z:zettels?tag = tag1 | tag2]]
2) Backlink filter: [[z:zettels?tag=tag1&backlinkfilter=pagetitle]]
3) Tag alias:  [[z:zettels?tag=tag1&alias ='My Notes']]

**Comment by srid** [comment]
See https://github.com/srid/neuron/issues/358
